vpp-manager/config/config.go

```diff
@@ -51,6 +49,24 @@ const (
 	DRIVER_VMXNET3         = "vmxnet3"
 )
 
+var Info = &VppManagerInfo{}
+
+type Link struct {
+	Name string
+	MTU  int
+}
+
+type VppManagerInfo struct {
+	Uplinks          map[int]int //map tapswifindex to linuxmtu
+	Status           int
```

Comment:
instead of an int `Status` could maybe be something more human-readable e.g. `ready` / `starting` the previous 0/1 was not super clear